_api-reference/validate.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: default
+title: Validate query
+nav_order: 87
+---
+
+# Validate query
+
+The Validate Query API checks if a large query will run without running the query. The query can be sent either as a path parameter or in the request body.
+
+## Paths and HTTP Methods
+
+The Validate Query API contains the following paths:
```

Comment:
Is this text necessary? The template doesn't have an intro sentence.

Reply:
It isn't. I'll delete this one.